cookbook/request/load_balancer_reverse_proxy.rst

```diff
@@ -95,6 +98,27 @@ That's it! It's critical that you prevent traffic from all non-trusted sources.
 If you allow outside traffic, they could "spoof" their true IP address and
 other information.
 
+My Reverse Proxy sends X-Forwarded-For but does not filter the Forwarded header
```

Comment:
We use title case in the Symfony docs. This means that this headline has to be capitialized like `My Reverse Proxy Sends X-Forwarded-For but does not Filter the Forwarded Header`